ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/priority-fee-input/index.scss

```diff
@@ -1,4 +1,4 @@
 .priority-fee-input {
-  border-top: 1px solid var(--ui-grey);
+  border-top: 1px solid var(--color-border-muted);
```

Comment:
If this is an input I think it should use `var(--color-border-default)`? `var(--color-border-muted)` should be used for low priority borders such as dividers where the user doesn't interact with them.